Makefile

```
@@ -275,7 +277,7 @@ ifneq ($(wildcard $(CURDIR)/mets.xml),)
 space = $() $()
 comma = ,
 define toolrecipe =
-$(TOOL) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
+$(TOOL) -l $(LOGLEVEL) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
```

The command-line `-l` is an _override_ to all logging defaults and config settings, so I cannot use this verbatim. 

```suggestion
$(TOOL) $(and $(LOGLEVEL),-l $(LOGLEVEL)) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
```

Okay?

---

Sure

---

`$(and $(LOGLEVEL),-l $(LOGLEVEL)` is a really elegant idiom for Makefiles btw.